Edit introduction to avoid repetition and add reference to TR-03111 (#27)
openpgp-pqc/draft-ehlen-openpgp-nist-bp-comp: draft-ehlen-openpgp-nist-bp-comp.md

# Introduction

> This document extends [draft-ietf-openpgp-pqc-12], which introduces post-quantum cryptography in OpenPGP based on hybrid KEMs and digitial signatures by using ML-KEM and ML-DSA with ECC algorithms using the Edwards Curves defined in {{RFC8032}} and {{RFC7748}}.

Reviewer: ```suggestion
This document extends [draft-ietf-openpgp-pqc-12], which introduces post-quantum cryptography in OpenPGP using hybrid KEMs and digital signatures that combine ML-KEM and ML-DSA with ECC algorithms based on the Edwards Curves defined in {{RFC7748}} and {{RFC8032}}.
```

Author: ```suggestion
This document extends [draft-ietf-openpgp-pqc-12], which introduces post-quantum cryptography in OpenPGP using hybrid KEMs and digital signatures combining ML-KEM and ML-DSA with ECC algorithms using the Edwards Curves defined in {{RFC8032}} and {{RFC7748}}.
```

I think that "which" instead of "that" would be correct but this would again be a bit repetitive and so I propose to use "combining".

> This document extends [draft-ietf-openpgp-pqc-12], which introduces post-quantum cryptography in OpenPGP based on hybrid KEMs and digitial signatures by using ML-KEM and ML-DSA with ECC algorithms using the Edwards Curves defined in {{RFC8032}} and {{RFC7748}}.

Author: I prepared a new version trying to combine your comment and Stavros' edit above.
